src/terms/xc.jl

```diff
@@ -40,6 +43,10 @@ function ene_ops(term::XcTerm, ψ, occ; ρ, kwargs...)
         dVol = basis.model.unit_cell_volume / prod(basis.fft_size)
         E += sum(Epp .* ρ.real) * dVol
     end
+    if term.scaling != 1
```

Comment:
Is this needed? Doesn't Julia optimise this away by itself?

Reply:
It doesn't, because it's not clever enough. In fact even `x .= x` is O(n). Also there is trickery afoot with floating point numbers which makes this kind of optimization difficult (eg `0.0 + x` is not always `x` when x is `-0.0`)

Reply:
Sorry `x .= x` is fast.